src/Hint/Naming.hs

```diff
@@ -49,9 +49,9 @@ namingHint :: DeclHint
 namingHint _ modu = naming $ Set.fromList $ concatMap getNames $ moduleDecls modu
 
 naming :: Set.Set String -> Decl_ -> [Idea]
-naming seen x = [suggestN "Use camelCase" x2 (replaceNames res x2) | not $ null res]
```

Comment:
In CI this file gives an error about Replace and Bind not being in scope. How come? Did you forget to commit some imports?

Reply:
Yes. Good spot :) will recommit with the new import.